### PR TITLE
Create release notes for 0.22.0

### PR DESCRIPTION
**Pull request by pmhayward**
Add doc/release-notes/0.22.md

Signed-off-by: Peter Hayward <pmhayward@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
@pshipton are you good with these.

File: doc/release-notes/0.22/0.22.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/10217">#10217</a></td>
+<td valign="top">Methods in <tt>com.ibm.lang.management.MemoryMXBean</tt> deprecated and replaced</td>
+<td valign="top">All versions</td>
+<td valign="top">The methods <tt>com.ibm.lang.management.MemoryMXBean.getGCMasterThreadCpuUsed()</tt> and <tt>com.ibm.lang.management.MemoryMXBean.getGCSlaveThreadsCpuUsed()</tt> are deprecated for removal in Java 16. The recommended methods to be used are <tt>com.ibm.lang.management.MemoryMXBean.getGCMainThreadCpuUsed()</tt> and <tt>com.ibm.lang.management.MemoryMXBean.getGCWorkerThreadsCpuUsed()</tt> respectively.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/9489">#9489</a></td>
+<td valign="top"><tt>java.lang.System.mapLibraryName()</tt> string suffix</tt></td>
+<td valign="top">From OpenJDK15, on AIX&reg;</td>
+<td valign="top"><tt>java.lang.System.mapLibraryName(libname)</tt> returns a representation of a native library in a platform-specific string with a <tt>.so</tt> suffix.</td>
```
Comment (pshipton) [inline doc/release-notes/0.22/0.22.md:75]: This should mention mapLibraryName returns `.a` prior to Java 15 and is changed to `.so` from Java 15.

**Review by dsouzai — APPROVED** [review]

**Review by mpirvu — APPROVED** [review]

File: doc/release-notes/0.22/0.22.md (hunk 1 above)
Comment (pmhayward) [reply doc/release-notes/0.22/0.22.md]: Explanation expanded to mention `.a` prior to 15

File: doc/release-notes/0.22/0.22.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/10217">#10217</a></td>
+<td valign="top">Methods in <tt>com.ibm.lang.management.MemoryMXBean</tt> deprecated and replaced</td>
+<td valign="top">All versions</td>
+<td valign="top">The methods <tt>com.ibm.lang.management.MemoryMXBean.getGCMasterThreadCpuUsed()</tt> and <tt>com.ibm.lang.management.MemoryMXBean.getGCSlaveThreadsCpuUsed()</tt> are deprecated for removal in Java 16. The recommended methods to be used are <tt>com.ibm.lang.management.MemoryMXBean.getGCMainThreadCpuUsed()</tt> and <tt>com.ibm.lang.management.MemoryMXBean.getGCWorkerThreadsCpuUsed()</tt> respectively.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/9489">#9489</a></td>
+<td valign="top"><tt>java.lang.System.mapLibraryName()</tt> string suffix</tt></td>
+<td valign="top">From OpenJDK15, on AIX&reg;</td>
+<td valign="top"><tt>java.lang.System.mapLibraryName(libname)</tt> returns a representation of a native library in a platform-specific string with a <tt>.so</tt> suffix. (Prior to OpenJDK15, returns from <tt>java.lang.System.mapLibraryName(libname)</tt> had a <tt>.a</tt> suffix.)</td>
```
Comment (pshipton) [inline doc/release-notes/0.22/0.22.md:75]: Should it be `have` instead of `had`? We still build and ship jdk8 and jdk11 with the `.a` behvaior.

Comment (pmhayward) [reply doc/release-notes/0.22/0.22.md]: Updated (and simplified!)

**Review by pshipton — APPROVED** [review]

**Pull request merged**